### ISSUE TITLE
[NEW] Simple SMS Messenger

### ISSUE THREAD

**Issue by MXC48** [tycrek/degoogle#456]
### Checklist

- [X] **I have read the guidelines in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md)**
- [X] Include my name in [CONTRIBUTORS.md](../blob/master/CONTRIBUTORS.md)
- [ ] I am affiliated with this alternative

### Alternative name

Simple SMS Messenger

### Section

Mobile applications

### Description

An easy and quick way of managing SMS and MMS messages without ads.

### URL

https://simplemobiletools.com/

### Repository

https://github.com/SimpleMobileTools/Simple-SMS-Messenger

### F-Droid ID

com.simplemobiletools.smsmessenger

### Is this alternative hosted in a country that performs surveillance?

- [ ] 5 Eyes
- [ ] 9 Eyes
- [ ] 14 Eyes

### Is there a cost to use this alternative?

Free

### Please indicate if self-hosting is possible, not possible, or the only option

Not possible

**Comment by dngray** [comment]
I would not add this. It uses no E2EE, which means your carrier and government get a copy of everything.

**Comment by Shahin-rmz** [comment]
> I would not add this. It uses no E2EE, which means your carrier and government get a copy of everything.

till now Signal is best option right?

**Comment by MXC48** [comment]
> > I would not add this. It uses no E2EE, which means your carrier and government get a copy of everything.
> 
> till now Signal is best option right?

Not necessarily, signal asks to register a phone number to use the application. For people who want to stay unidentified se is not the best option and then many people like me do not trust signal.

**Comment by Shahin-rmz** [comment]
> many people like me do not trust signal.

Any good suggestion out there?

**Comment by dngray** [comment]
For SMS apps, not using SMS for a start. As I said in https://github.com/tycrek/degoogle/issues/468

> This does not employ any end-to-end encryption, so it really has nothing to do with "degoogling".

> Ironically you're actually better off with [Google Messenger](https://www.gstatic.com/messages/papers/messages_e2ee.pdf) these days, because it does support [Signal Protocol](https://en.wikipedia.org/wiki/Signal_Protocol) over [RCS](https://en.wikipedia.org/wiki/Rich_Communication_Services), which means you get E2EE that Google cannot read anyway. It is worth noting though Signal is still a better option as there is less metadata, particularly with [sealed sender](https://signal.org/blog/sealed-sender/).
